src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,24 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
```

Comment:
The logger is initialized correctly, but ensure that the logger configuration includes appenders for both File and Console as required by the task description.